include/envoy/common/http/BUILD

```diff
@@ -0,0 +1,14 @@
+load(
+    "//bazel:envoy_build_system.bzl",
+    "envoy_cc_library",
+    "envoy_package",
+)
+
+licenses(["notice"])  # Apache 2
+
+envoy_package()
+
+envoy_cc_library(
+    name = "alternate_protocols_cache_interface",
```

Comment:
generally we skip interface for these library names

Reply:
-> include/envoy/http and I'm so sorry if I guessed the wrong name offhand

Reply:
Moved to envoy/http/. But the others in that dir have _interface so I kept the suffix.